docs/release-notes/new-experiment-list.rst

```diff
@@ -0,0 +1,14 @@
+:orphan:
+
+**New Features**
+
+This release includes a new experiment list page. In addition to general improved performance and stability we've added a number of new features.
```

Comment:
WebUI: In addition to improved performance and stability, we've added the following features to the experiment list page: